package/docker/Dockerfile.gotools

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Authors: Gaulthier Gain <gaulthier.gain@uliege.be>
+#
+# Copyright (c) 2020, Université de Liège., ULiege. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+ARG GOLANG_VERSION=1.14.4
+ARG SHARED_DIR=/usr/shared
+
+FROM golang:${GOLANG_VERSION}
+
+RUN mkdir /usr/src/gotools
+COPY gotools/tools_Linux /usr/src/gotools
+WORKDIR /usr/src/gotools
+
+RUN mkdir /usr/shared
+VOLUME [${SHARED_DIR}]
+
+RUN set -xe
+RUN chmod +x ./tools_Linux
```

Comment:
Likely this file can be put into the Dockerfile verbatim as it won't be run outside

Reply:
I've  just seen this is a binary.  The Dockerfile is intended to build go files?

Reply:
What do you mean by binary? Creating the binary from sources only in the Dockerfile ?

Reply:
yes